servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -565,16 +568,16 @@ void RendererSceneRenderRD::_render_buffers_post_process_and_tonemap(const Rende
 			dest_fb = texture_storage->render_target_get_rd_framebuffer(render_target);
 		}
 
-		tone_mapper->tonemapper(internal_texture, dest_fb, tonemap);
+		post_processor->postprocessor(internal_texture, dest_fb, post_process_settings);
 
 		RD::get_singleton()->draw_command_end_label();
 	}
 
 	if (fsr && can_use_effects && rb->get_scaling_3d_mode() == RS::VIEWPORT_SCALING_3D_MODE_FSR) {
 		RD::get_singleton()->draw_command_begin_label("FSR 1.0 Upscale");
-
+		printf("FSR ?");
```

Comment:
FSR upscaling should still take place when using viewport mode 3D only. As users will expect a full resolution buffer

Reply:
IMO render resolution is expected in this case.